support-frontend/build.sbt

```diff
@@ -93,4 +93,6 @@ javaOptions in Universal ++= Seq(
 
 javaOptions in Test += "-Dconfig.file=test/selenium/conf/selenium-test.conf"
 
+scalacOptions += "-Ypartial-unification"
```

Comment:
needed so we can sequence an Either or any other higher order operation on something with more than one type parameter. actually I refactored the code that needed it out, but this is good to have on anyway
explanation of what it is here:
https://gist.github.com/djspiewak/7a81a395c461fd3a09a6941d4cd040f2

Reply:
I'd leave it out if it's not needed for this PR just to keep things simple

Reply:
sounds sensible, done that now.